src/deploy.js

```diff
@@ -112,7 +113,7 @@ const deploy = async (opts) => {
   } = process.env;
 
   console.log(
-    `info: publishing app ${REACT_APP_GRAASP_APP_ID} version ${REACT_APP_VERSION}`,
+    `publishing app ${REACT_APP_GRAASP_APP_ID} version ${REACT_APP_VERSION}`,
   );
 
   // configure the deployment
```

Comment:
@ugGit, I think that you can use this promise version of getting the credentials.

https://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/Credentials.html#getPromise-property